zng/docs/spec.md

```diff
@@ -652,12 +680,13 @@ Type | Format
 `bstring` | a UTF-8 string with `\x` escapes of non-UTF binary data
 `enum` | a string representing an enumeration value defined outside the scope of ZNG
 `ip` | a string representing an IP address in [IPv4 or IPv6 format](https://tools.ietf.org/html/draft-main-ipaddr-text-rep-02#section-3)
+`port` | a string representing an 16-bit network port |
```

Comment:
Nit: `s/decimal string representation of an unsigned, 16-bit integer/` for consistency with other integer types.

Reply:
Fine point. While I was thinking about consistency, I noticed another entry in the same table led with `a decimal` so I shortened that one to just `decimal` to be like the others. Then I noticed the whole table was lacking in spaces to make it pretty at the Markdown level, so I tidied that up a bit while I was at it.